Node.js/GettingStarted/azure_batch_client.js

```diff
@@ -0,0 +1,196 @@
+// Initializing node.js modules
+
+var batch = require('azure-batch');
+var https = require('https');
+
+var fs = require('fs');
+var uuid = require('node-uuid');
+
+var accountName = '<azure-batch-account-name>';
+var accountKey = '<account-key-downloaded>';
+var accountUrl = '<account-url>'
+// Setting up Job configuration along with preparation task
+var jobId = "processcsvjob"; 
+
+
+// Create Batch credentials object using account name and account key
+ var credentials = new batch.SharedKeyCredentials(accountName,accountKey);    
+
+ // Create Batch service client
+
+var batch_client = new batch.ServiceClient(credentials,accountUrl);
+
+var now = new Date();
+
+
+var poolid = "processcsv_"+ now.getFullYear() + now.getMonth() + now.getDay() + now.getHours() + now.getSeconds()
```

Comment:
Maybe not an issue, I think this will produce IDs that are a bit hard to relate back to timestamps - if you want something that relates back to a timestamp then consider zero-padding to get a YYYYMMDDhhmmss format.  If you don't want something that relates back to a timestamp then consider using a GUID instead.